### ISSUE TITLE
survey complete = yes doesn't happen on all devices

### ISSUE THREAD

**Issue by thibaultmol** [pietervdvn/MapComplete#1251]
I know the survey is just a temp thing, but just wanted to mention that I noticed I clicked the button my phone but on desktop it keeps asking about it (so I now also clicked it there). But I guess maybe this could be included in the user settings so if it's clicked on one device, it's marked as completed on all other devices?

Or is this on purpose? (I can see both cases)

**Comment by pietervdvn** [comment]
I'm not entirely sure what you mean, but I think that: if you skip all questions to the end, that this isn't reflected on other devices or upon reloading the page? And thus, previously skipped questions are asked again?

First of all, there is no 'survey=complete' tag which is placed upon elements. Being surveyed is not an inherent property, especially as the questions asked my change; or the surveyor might not fill in certain questions.

Second, putting a marker into the user preferences to 'sync' this 'skipped all remaining questiosn'-issue would be a _huge_ abuse of the preferences-system (which I'm already abusing a little bit).

Third: seeing the question again on desktop might trigger answering it after all ;)

In other words, I do think this is a duplicate of #256 . If you feel this is not the case, let me know.
